src/elm/Page/Transfer.elm

```diff
@@ -0,0 +1,414 @@
+        [ a
+            [ class "items-center flex absolute"
+            , Route.href Route.Dashboard
+            ]
+            [ Icons.back ""
+            , p [ class "text-white text-sm ml-2" ]
+                [ text (I18Next.t shared.translations "back")
+                ]
+            ]
+        , p [ class "text-white mx-auto" ] [ text (t "transfer_result.title") ]
+        ]
+
+
+viewDoggo : LoggedIn.Model -> Transfer -> State -> Html Msg
+viewDoggo loggedIn transfer state =
+    let
+        t =
+            I18Next.t loggedIn.shared.translations
+    in
+    div [ class "static flex" ]
+        [ div [ class "w-full bg-green h-50" ]
+            [ div [ class "flex-row" ]
+                [ div [ class "px-4 py-2 m-2" ]
+                    [ h2 [ class "text-center mt-8 font-medium font-sans text-white not-italic" ]
+                        [ text <|
+                            case state of
+                                Transferred ->
+                                    t "transfer_result.transfer_success"
+
+                                Received ->
+                                    t "transfer_result.receive_success"
+
+                                NotInvolved ->
+                                    t "transfer_result.transfer_success"
+                        ]
+                    ]
+                , div [ class "bg-no-repeat bg-auto h-64 ml-32 -mt-5 px-4 py-2 m-2 items-center mt-5 justify-center", style "background-image" "url(/images/transfer-doggo.svg)" ]
+                    []
+                , div [ class "flex sm:justify-center sm:items-center -mt-16 absolute md:w-full md:mx-auto  sm:right-0" ]
+                    [ viewTransferCard loggedIn transfer state
+                    ]
+                ]
+            ]
+        ]
+
+
+viewTransferCard : LoggedIn.Model -> Transfer -> State -> Html Msg
+viewTransferCard loggedIn transfer state =
+    let
+        ipfsUrl =
+            loggedIn.shared.endpoints.ipfs
+    in
+    div [ class "flex flex-row block rounded overflow-x-scroll sm:overflow-hidden bg-gray-100" ]
+        [ div [ class "sm:px-4 sm:py-2 sm:m-2 p-0" ]
+            [ div [ class "h-8 w-8 rounded-full mx-auto mt-5" ]
+                [ Avatar.view ipfsUrl
+                    (case state of
+                        Received ->
+                            transfer.to.avatar
+
+                        Transferred ->
+                            transfer.from.avatar
+
+                        NotInvolved ->
+                            transfer.from.avatar
+                    )
+                    "h-10 w-10"
+                ]
+            , div [ class "px-6 py-4" ]
+                [ span [ class "text-base inline-block bg-black rounded-full px-3 py-1 text-sm font-semibold text-white" ]
+                    [ case state of
+                        Received ->
+                            text "You"
+
+                        Transferred ->
+                            text "You"
```

Comment:
You can refactor this since the text is the same in both `Received` and `Transferred`